apps/nextra/pages/en/build/cli/start-from-template.mdx

```diff
@@ -0,0 +1,44 @@
+import { Card, Cards, RemoteCodeblock, permalinkFetch } from '@components/index';
+import { Steps } from "nextra/components"
+
+export async function getStaticProps() {
+  return await permalinkFetch([
+    "https://github.com/aptos-labs/aptos-core/blob/77e1d222ebc5e7294e115e0d090c001da1d0e072/aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move#L1-L59"
```

Comment:
You're going to want this one actually: https://github.com/aptos-labs/aptos-core/blob/afd3706c17bcccfb39a9d6059aecbfa648ed295d/aptos-move/move-examples/hello_blockchain/sources/hello_blockchain.move#L4